IPs/RSKIP188.md

```diff
@@ -0,0 +1,95 @@
+# Title
```

Comment:
Title "Precompile for BLS12-381 curve operations" must be in this first line also.

Reply:
done :D